src/builddocs.jl

```diff
@@ -99,13 +99,14 @@ end
 addentry!(dict, object, entry) = setindex!(dict, entry, object)
 
 function processcomment(meta, state, file, block)
-    mac, line, _ = block
+    mac, line, expr = block
+    code = expr
 
     source = (linenumber(line), file)
     docs   = Docs{meta.data[:format]}(exec(state, mac))
 
     object = Comment()
-    entry  = Entry{:comment}(meta.modname, source, docs)
+    entry  = Entry{:comment}(meta.modname, source, code, docs)
 
     object, entry
 end
```

Comment:
Watch out with the unquoted macro names :) I'm sure doc doesn't want to be pinged on this discussion, sorry about that doc.